spec/models/study_spec.rb

```diff
@@ -663,10 +663,11 @@
 
         it 'does not error if the data_release_timing is invalid' do
```

Comment:
hmm 🤔 Does it assert for raising _any_ error or a specific error? It looks like it's trying to check if the record is valid or not?